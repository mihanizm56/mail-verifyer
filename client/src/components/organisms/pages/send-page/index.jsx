import React from "react";
import { Sender } from "../../../../modules";
import "./send-page.css";

export const SendPage = () => (
	<div className="send-page">
		<Sender />
	</div>
);
