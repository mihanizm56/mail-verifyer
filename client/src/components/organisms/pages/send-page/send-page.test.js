import React, { createElement } from "react";
import { shallow } from "enzyme";
import { SendPage } from "../send-page";

describe("test SendPage", () => {
	beforeEach(() => {
		jest.resetModules();
		jest.clearAllMocks();
	});

	describe("test render contract", () => {
		it("render snap with all props", () => {
			const component = shallow(<SendPage />);

			expect(component).toMatchSnapshot();
		});
		it("render Sender module", () => {
			const component = shallow(<SendPage />);
			const Sender = component.find("Sender");

			expect(Sender.length).toEqual(1);
		});
	});
});
