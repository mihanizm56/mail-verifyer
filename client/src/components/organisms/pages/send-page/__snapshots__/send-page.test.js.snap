// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test SendPage test render contract render snap with all props 1`] = `
<div
  className="send-page"
>
  <Sender />
</div>
`;
